lvmreport/reader.go

```diff
@@ -34,14 +32,26 @@ func (r *reader) Decode() {
 
 	var data root
 
-	if err := r.dec.Decode(&data); err != nil {
+	// Workaround for LVM bug which can add escaped null literals (`\0`) in strings
+	// This is invalid JSON, so we fix it by just removing all escaped nulls
+	rawData, err := io.ReadAll(r.innerReader)
+	if err != nil {
+		r.err = fmt.Errorf("reading src data failed: %w", err)
+	}
+
+	fixedRawData := strings.ReplaceAll(string(rawData), "\\0", "")
```

Comment:
Use `bytes.ReplaceAll` (https://pkg.go.dev/bytes#ReplaceAll) instead of converting to/from `string`.